src/main/java/io/vertx/jdbcclient/impl/actions/JDBCPropertyAccessor.java

```diff
@@ -0,0 +1,35 @@
+package io.vertx.jdbcclient.impl.actions;
```

Comment:
For Eclipse Compliance, can you add a copyright header just like the other files?